app/controllers/teams_controller.rb

```diff
@@ -26,6 +26,7 @@ def edit
   # POST /users.json
   def create
     @team = Team.new(team_params)
+    @team.active = true
```

Comment:
All of this is fine, but consider two things:

Merge, create!, rescue 
```ruby
@team = Team.create!(team_params).merge(active: true)
   format.html....
rescue
  format.html { render :new }
  format.json { render json: @team.errors, status: :unprocessable_entity }	         format.json { render json: @team.errors, status: :unprocessable_entity }
```

Merge allows you to add other params to the hash.
`create!` will fail loudly in your logs
[Check out rails rescue logic. ](https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b)

Rescue allows you to pass an exception as well if needed.